docs/master/api-reference/directives.md

```diff
@@ -1111,6 +1111,47 @@ class PlacedOrder
 }
 ```
 
+## @extends
```

Comment:
`@extends` would be the most natural name, but it is already used by Apollo Federation, which Lighthouse supports. How about `@inherits(from: "Foo")`?

Reply:
True.

I think the default extends adds certain attributes to an already existing type. 
🤔

Reply:
That is not at all what `@extends` does in the context of federation. We don't have to discuss that here though; I am just saying that the name is taken.

Reply:
Sorry meant extend.

Reply:
Oh, yeah that is right. Another good reason to choose a different name for this.